resources/js/app.js

```diff
@@ -1108,3 +1108,120 @@ $(document).on("focusin", function(e) {
 		  e.stopImmediatePropagation();
 	  }
 });
+/*
+   * Finance - Invoices module JS code
+   */
+function getcurrency() {
+	var currency = document.getElementById("currency").value;
+	return currency;
+}
+function number2text(value) {
+	var curr = getcurrency();
```

Comment:
This is more meaningful
```suggestion
	var currency = getcurrency();
```